tests/install_upgrade_operators/conftest.py

```diff
@@ -39,6 +39,18 @@
 LOGGER = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="session")
+def validate_cluster_not_tainted(admin_client, hco_namespace):
```

Comment:
fixture names should be nouns and not verbs; i.e what is the outcome of the fixture or the required setup for the test 
in this case could be `is_cluster_tainted` or `untainted_cluster`